assets/styles/global/_tooltip.scss

```diff
@@ -134,7 +134,6 @@
 .tooltip, .popover {
   &[aria-hidden='true'] {
     visibility: hidden;
-    display: none !important;
```

Comment:
@westlywright Will you take a look at this one? I removed this because it was causing a problem with the positioning the of the tooltip on the first render. 

If you need that we may need to find a workaround because it was preventing v-tooltip from seeing the dimensions of the tooltip and therefore it couldn't center it properly.

![hover-bug](https://user-images.githubusercontent.com/55104481/89841957-250d6b80-db29-11ea-8abc-7b99fed44971.gif)

Reply:
I spoke to @westlywright about this. Without `display: none` ButtonDropdown would render larger than intended when SSR was enabled. 

We found a workaround with `position: absolute` as it removed it from the layout of the initial render but still allowed v-tooltip to see the dimensions.